specification/syntex/resource-manager/Microsoft.Syntex/preview/2022-09-15-preview/syntex.json

```diff
@@ -0,0 +1,521 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "version": "2022-09-15-preview",
+    "title": "Syntex",
+    "description": "Syntex provider"
```

Comment:
So I notice there is this company called Syntex... is this going to be a potential trademark issue?

Reply:
Actually I guess its defunct now... I guess that should eliminate the issue but maybe not completely a potential association in people's minds.

Reply:
The other thing that commonly comes up is that brand names are always changing even at MS. Would it be better to use something generic that won't need rebranding later?

Reply:
Syntex is also our brand, we've been working on this provider for a while:
https://www.microsoft.com/en-us/microsoft-365/enterprise/sharepoint-syntex-overview

Reply:
The decision to name provider Syntex was made more than a year ago, and we already have private preview customers behind AFEC protection.  We need remove AFEC protection by Ignite, so we have to keep the name as is.